src/Arcus.Observability.Telemetry.AspNetCore/Extensions/ILoggerExtensions.cs

```diff
@@ -1,16 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Globalization;
 using Arcus.Observability.Telemetry.Core;
 using GuardNet;
 using Microsoft.AspNetCore.Http;
 
 // ReSharper disable once CheckNamespace
 namespace Microsoft.Extensions.Logging
 {
+    /// <summary>
+    /// Telemetry extensions on the <see cref="ILogger"/> instance to write Application Insights compatible log messages.
+    /// </summary>
     public static class ILoggerExtensions
     {
-        // <summary>
+        private static string InvariantDateTimeWithMillisecondsFormat = "yyyy-MM-ddTHH:mm:ss.fffffff zzz";
```

Comment:
Since we use it multiple times we should maybe move it to a shared place?

Reply:
Yes, I was thinking of that as well.   Problem is where to put it ?  I initially thought to put it in Telemetry.Core, but Telemetry.AspNetCore has no dependency on Telemetry.Core, and I thought it would maybe be overkill to create that dependency, but we can do that

Reply:
Let's do that!

Reply:
My bad, the dependency was already there.

Reply:
Done.